blocks/init/src/Blocks/components/accordion/accordion.php

```diff
@@ -36,7 +36,7 @@
 
 <div
 	class="<?php echo \esc_attr("{$accordionClass}"); ?>"
-	data-accordion-open="<?php echo $accordionIsOpen ? 'true' : 'false'; ?>"
+	data-accordion-open="<?php echo \esc_attr($accordionIsOpen ? 'true' : 'false'); ?>"
```

Comment:
This should be fixed upstream in coding standards. We shouldn't need to escape this since it will always echo a constant string (I believe previously this was detected by PHPCS and no error was thrown).

Reply:
Actually no, because the `$accordionIsOpen` comes from line 25 from `Components::checkAttr`, which is not safe output.